packages/cardpay-sdk/sdk/reward-pool/base.ts

```diff
@@ -241,19 +228,6 @@ export default class RewardPool {
     return leafs;
   }
 
-  async rewardTokenBalance(
-    address: string,
-    tokenAddress: string,
-    rewardProgramId?: string
-  ): Promise<RewardTokenBalance> {
-    let balance = await this.getBalance(address, tokenAddress, rewardProgramId);
-    return {
-      rewardProgramId,
-      tokenAddress,
-      balance,
-    };
-  }
-
```

Comment:
Remove `rewardTokenBalance` it was confusing. Case covered by `rewardTokenBalances`